src/TimerRecordDialog.cpp

```diff
@@ -591,9 +591,21 @@ int TimerRecordDialog::ExecutePostRecordActions(bool bWasStopped) {
 
    // Do Automatic Export?
    if (m_bAutoExportEnabled) {
+      const auto& tracks = TrackList::Get(mProject);
       Exporter e{ mProject };
+      
+      bool skipSilenceAtBeginning;
+      gPrefs->Read(wxT("/AudioFiles/SkipSilenceAtBeginning"),
```

Comment:
Another opportunity to use `BoolSetting`.

But be careful:  this one is used as an argument to `TieCheckBox`.

When that happens, you also need to use `Invalidate` in the `Commit()` member function of the preference panel.

I intend to remove that need for `Invalidate`, but it requires first using a named Setting object in every call of the `Tie...` functions, in Preferences, and also in Export dialogs, and also in SoundActivatedRecord.cpp